controllers/apis/auth.go

```diff
@@ -0,0 +1,47 @@
+package apis
```

Comment:
Similarly to my previous comment, how about:

```suggestion
package apicontrollers
```